localstack/services/ec2/provider.py

```diff
@@ -381,6 +395,67 @@ def modify_launch_template(
 
         return result
 
+    @handler("DescribeVpcEndpointServices", expand=False)
+    def describe_vpc_endpoint_services(
+        self,
+        context: RequestContext,
+        request: DescribeVpcEndpointServicesRequest,
+    ) -> DescribeVpcEndpointServicesResult:
+        ep_services = VPCBackend._collect_default_endpoint_services(
+            account_id=context.account_id, region=context.region
+        )
+
+        moto_backend = get_moto_backend(context)
+        service_names = [s["ServiceName"] for s in ep_services]
+        execute_api_name = f"com.amazonaws.{context.region}.execute-api"
+
+        if execute_api_name not in service_names:
+            # ensure that the service entry for execute-api exists
+            service_names.append(execute_api_name)
```

Comment:
nit: why do we append the `execute_api_name` here in a list that is dynamically created on every API call?

Reply:
Great catch, thanks - that is totally not necessary here. (Will remove..)